examples/python/human_pose_tracking/README.md

```diff
@@ -16,9 +17,227 @@ channel = "main"
+
+```python
+def track_pose(video_path: str, *, segment: bool, max_frame_count: int | None) -> None:
+    mp_pose = mp.solutions.pose
+
+    rr.log(
+        "/",
+        rr.AnnotationContext(
+            rr.ClassDescription(
+                info=rr.AnnotationInfo(id=0, label="Person"),
+                keypoint_annotations=[rr.AnnotationInfo(id=lm.value, label=lm.name) for lm in mp_pose.PoseLandmark],
+                keypoint_connections=mp_pose.POSE_CONNECTIONS,
+            )
+        ),
+        timeless=True,
+    )
+```
+
+A timeless
+[`ClassDescription`](https://www.rerun.io/docs/reference/types/datatypes/class_description) is logged (note, that
+this is equivalent to logging an
+[`AnnotationContext`](https://www.rerun.io/docs/reference/types/archetypes/annotation_context) as in the
+segmentation case). 
+
+## Body Pose Landmarks as 2D Points
+
+[![Body Pose Landmarks_2d_points](https://github.com/rerun-io/rerun/assets/49308613/d5f1b3b5-c55e-44a3-8ad2-6e425ae8e627)](https://github.com/rerun-io/rerun/assets/49308613/b7154548-f5ab-4371-b677-0c902404630f)
+
+You can extract body pose landmark points as image coordinates. 
+These coordinates are then logged as 2D points using the  type [`Points2D`](https://www.rerun.io/docs/reference/types/archetypes/points2d).
+
+```python
+def track_pose(video_path: str, *, segment: bool, max_frame_count: int | None) -> None:
+    mp_pose = mp.solutions.pose
+    
+    rr.log(
+        "/",
+        rr.AnnotationContext(
+            rr.ClassDescription(
+                info=rr.AnnotationInfo(id=0, label="Person"),
+                keypoint_annotations=[rr.AnnotationInfo(id=lm.value, label=lm.name) for lm in mp_pose.PoseLandmark],
+                keypoint_connections=mp_pose.POSE_CONNECTIONS,
+            )
+        ),
+        timeless=True,
+    )
+
+    with closing(VideoSource(video_path)) as video_source, mp_pose.Pose(enable_segmentation=segment) as pose:
+        for idx, bgr_frame in enumerate(video_source.stream_bgr()):
+            if max_frame_count is not None and idx >= max_frame_count:
+                break
+
+            rgb = cv2.cvtColor(bgr_frame.data, cv2.COLOR_BGR2RGB)
+            
+            # Associate frame with the data
+            rr.set_time_seconds("time", bgr_frame.time)
+            rr.set_time_sequence("frame_idx", bgr_frame.idx)
+            
+            # Present the video
+            rr.log("video/rgb", rr.Image(rgb).compress(jpeg_quality=75))
+
+            # Get the prediction results
+            results = pose.process(rgb)
+            h, w, _ = rgb.shape
+            
+            # Log 2d points to 'video' entity
+            landmark_positions_2d = read_landmark_positions_2d(results, w, h)
+            if landmark_positions_2d is not None:
+                rr.log(
+                    "video/pose/points",
+                    rr.Points2D(landmark_positions_2d, class_ids=0, keypoint_ids=mp_pose.PoseLandmark),
```

Comment:
I think `keypoint_ids=mp_pose.PoseLandmark` is the non-obvious part here that would be nice to highlight in the text before.